src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -54,6 +54,19 @@ define([
                         var saveCode = $(e.target).data('saveCode');
                         self.plugin.turnOff();
                         self.triggerEvent('launchpad:activate-scenario', saveCode);
+                    })
+                    .on('click', 'button.truncate-toggle', function (e) {
+                        var parentEl = $(e.target.parentElement);
+                        $('p', parentEl).each(function() {
+                            $(this).toggleClass('truncate-box');
+                        });
```

Comment:
I think the jQuery way of doing this would be something like `$el.find('p').toggleClass(...)`.